src/ESS.jl

```diff
@@ -1,33 +1,38 @@
-using MCMCDiagnosticTools
+import MCMCDiagnosticTools
 
 export relative_eff, psis_ess, sup_ess
 
 """
     relative_eff(
-        sample::AbstractArray{<:Real, 3}; 
-        method=MCMCDiagnosticTools.FFTESSMethod()
+        sample::AbstractArray{<:Real, 3};
+        source::Union{AbstractString, Symbol} = "default",
+        kwargs..., 
     )
 
-Calculate the relative efficiency of an MCMC chain, i.e. the effective sample size divided
+Calculate the relative efficiency of an MCMC chain, i.e., the effective sample size divided
 by the nominal sample size.
 
+If `lowercase(String(source))` is `"default"` or `"mcmc"`, the relative effective sample size is computed with `MCMCDiagnosticTools.ess`.
+Otherwise a vector of ones for each chain is returned.
+The remaining keyword arguments `kwargs` are forwarded to `MCMCDiagnosticTools.ess`.
+
 # Arguments 
 
-  - `sample::AbstractArray{<:Real, 3}`: An array of log-likelihood values.
+  - `sample::AbstractArray{<:Real, 3}`: An array of log-likelihood values of the shape `(parameters, draws, chains)`.
 """
 function relative_eff(
     sample::AbstractArray{<:Real,3}; 
-    source::Union{AbstractString, Symbol}="default", maxlag=size(sample, 2), kwargs...
+    source::Union{AbstractString, Symbol}="default",
+    kwargs...,
```

Comment:
Yes, maybe that's a better option. I mainly removed it because the default was not used anyway - due to https://github.com/TuringLang/MCMCDiagnosticTools.jl/blob/304ecb2f4d926239a678971662d4a367ddbd7be4/src/ess_rhat.jl#L473 it will be at most `size(sample, 2) - 4`.

Reply:
Yeah, but I think this is fine. The decrease at that line was added because it improves the estimate for antithetic chains with likely no real change to the estimate for non-antithetic chains. But for long and poorly mixing chains, reducing the maxlag from `size(sample, 2) ÷ 2 - 4` to `246` could give quite different estimates.

The counterargument is that one should only be performing LOO-CV if marginal diagnostics already indicate that the chains are well-mixing, in which case this choice should not matter much.